x-pack/plugins/monitoring/dev_docs/how_to/running_components_from_source.md

```diff
@@ -107,9 +107,9 @@ You'll likely want to reset the passwords for the secondary cluster as well:
 
 ```shell
 curl -k -u elastic:'PASSWORD' -H 'Content-Type: application/json' \
-  http://localhost:9210/_security/user/elastic/_password -d'{"password": "changeme"}'
+  http://localhost:9201/_security/user/elastic/_password -d'{"password": "changeme"}'
```

Comment:
It might be better to update line 103 instead.

My intent in https://github.com/elastic/kibana/pull/137548 was to make the single node "secondary" start at port 9210/9310.

This makes space for a multi-node primary in the 920X/930X space.

Reply:
If this is confusing we can go back to 9201/9301 and just have the multi-node setup be different.

Originally I was hoping we could "append" nodes to go from single to multi-node, but in practice it's easiest to just wipe and start over with the right set of seed hosts.

Reply:
Ah, that makes sense! I'll update that

Reply:
Updated